source/js/buyers-guide/search/pni-sort-dropdown.js

```diff
@@ -164,28 +164,55 @@ export class PNISortDropdown {
   /**
    * Focus on the next list item based on the given direction.
    *
-   * @param {number} direction - They key code of the direction to move the focus (e.g., key code for up arrow / down arrow)
-   *
-   * @todo Consider refactoring this method to make it more readable and maintainable.
+   * @param {number} direction The key code of the direction to move the focus (e.g., key code for up arrow / down arrow)
    */
   focusNextListItem(direction) {
```

Comment:
I refactored this function into new functions `focusNextItem`, `focusPreviousItem`, and `focusItem`